themes/ddbasic/sass/components/panel/class-panel.scss

```diff
@@ -0,0 +1,431 @@
+@import '../../base.scss';
```

Comment:
Describe component contents/purpose, why are these styles grouped? What is the common denominator?